examples/flutter_gallery/lib/demo/video_demo.dart

```diff
@@ -324,7 +332,9 @@ class _ConnectivityOverlayState extends State<ConnectivityOverlay> {
 
   @override
   void dispose() {
-    connectivitySubscription.cancel();
+    if (connectivitySubscription != null) {
+      connectivitySubscription.cancel();
```

Comment:
can be `connectivitySubscription?.cancel()`

Reply:
Ah, good point!

Reply:
Done.